workflow/scripts/osemosys_global/transmission/main.py

```diff
@@ -53,57 +76,80 @@ def main(
     
     # CALL FUNCTIONS
     
+    # Correct the input data from the Global Transmission Database (GTD).
+    gtd_exist_corrected, gtd_planned_corrected = correct_gtd_data(gtd_exist, 
```

Comment:
Is it easier to just return one dataframe? Or just move up the res capacity calc right under this to just have one transmission datastructure to pass around?

Reply:
Some of the functions that use `gtd_exist_corrected` and `gtd_planned_corrected` require it in the format as initially generated (e.g. with the from and to columns). The `res_capacity_transmission` output format is sligthly different. No biggy to change if necessary but the transmission scripts are quite quick as it is already so might not be worth the effort to reduce the nr. of dfs that are being passed around. I also find it convenient to have the existing/planned df's separately in case they need to be called in future functions. Let me know!

Reply:
Fair point and your right; not urgent and it works as is :)